subsys/bluetooth/controller/hal/nrf5/cntr.c

```diff
@@ -24,6 +24,9 @@ void cntr_init(void)
 			     RTC_EVTENSET_COMPARE1_Msk);
 	NRF_RTC->INTENSET = (RTC_INTENSET_COMPARE0_Msk |
 			     RTC_INTENSET_COMPARE1_Msk);
+#if defined(CONFIG_BOARD_NRF52_NWTSIM)
```

Comment:
Would it have been possible to use `if (IS_ENABLED(CONFIG_BOARD_NRF52_NWTSIM)` here and in most other places of the patch? I think that should generally be preferred since it results in slightly more readable code and the compiler will "see" the code in the branch regardless of the Kconfig setting, i.e. it'll be able to report typos, etc.

Reply:
I understand your point.
In this case, it would led to a lot of implicit function declaration warnings, as the prototypes of the functions in the ifdefs are not visible in real target compilation.
I guess you could then argue we could just add ifndef(CONFIG_BORD_..) defines to replace the prototypes.. (but I guess that would be the same as the current ifdefs)

This patch tried to minimize the amount of changes/additions to the real target code, hiding everything else in files only used and known about in the simulated target board.